api/modules/os.py

```diff
@@ -15,3 +15,13 @@
 
 See the `Starlark spec <https://github.com/bazelbuild/starlark/blob/master/spec.md#built-in-methods>`_ for more.
 """
+
+def getcwd() -> str:
+  """Returns a string representation of the current working directory.
+
+  If your Tiltfile runs any commands, they will inherit this working directory.
+
+  When your Tiltfile loads() another Tiltfile, the working directory will
```

Comment:
I see a couple potential ambiguities here:
1. This sounds kind of like a state change, like it might set the expectation that `load('foo/bar'); print(os.getcwd())` would return 'foo/bar' rather than '.' (or whatever).
2. (I'm assuming) this returns the Tiltfile's directory, not the CWD of the calling process.

Maybe something like this?

> The working directory for the currently executing Tiltfile. If your Tiltfile runs any commands, they run from this directory.
> If this is called while `:meth:load`ing or `:meth:include`ing a Tiltfile, it will return the directory of the loaded/included Tiltfile.

or maybe #1 above is mostly addressed by changing "When" to "While".

Reply:
done! tweaked your language a bit to match the verb tenses in the rest of the doc